### PR TITLE
Feat: create accordion

### PR DESCRIPTION
**Pull request by atomictangerine**
## What I did

* Added the accordion component
* manually added the props table
* updated the README for the component
* added the accordion to the kitchen-sink page
* added a storybook example

### CODE REVIEW

**Comment by artmsilva** [comment]
Can you write interaction tests?

File: build.washingtonpost.com/docs/components/accordion.mdx [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+
+### Focus / Keyboard interaction
+
+The accordion component is accessible via keyboard interaction. The "tab" key can be used to navigate items from top to bottom. The item currently in focus is denoted by a blue _outline_, using the CSS outline property. Pressing the "enter" ("return") key will expand or collapse the item in focus.
+
+```jsx withPreview
+export default function Example() {
+  return (
+    <Box css={{ width: 500 }}>
+      <Accordion.Root type={ACCORDION_TYPE.single} defaultValue="item-1">
```
Comment (wp-aberg) [inline build.washingtonpost.com/docs/components/accordion.mdx:288]: I don't think we have an easy way to set the height of the demos, but with three items this one gets cut off and ends up looking odd. Can we go down to two?

image

File: build.washingtonpost.com/docs/components/accordion.mdx [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris in
+            augue in felis pharetra finibus. In sagittis aliquam augue. Lorem
+            ipsum dolor sit amet, consectetur adipiscing elit.
+          </Accordion.Content>
+        </Accordion.Item>
+      </Accordion.Root>
+
+      <Accordion.Root type={ACCORDION_TYPE.single} collapsible={true}>
+        <Accordion.Item value="item-1">
+          <Accordion.Trigger css={{ fontSize: theme.fontSizes[400] }}>
```
Comment (wp-aberg) [inline build.washingtonpost.com/docs/components/accordion.mdx:403]: Similar to focus, this huge font with a fixed demo height leads to some odd overlap with the warning, can we come down a bit?

image

File: ui/accordion/src/Accordion.tsx [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+type AccordionTriggerProps = React.ComponentProps<typeof AccordionTrigger>;
+AccordionTrigger.displayName = "AccordionComponentHeader";
+
+const StyledIcon = styled(Icon, {
+  marginLeft: theme.space[150],
+  minWidth: theme.fontSizes[100],
+});
+
+const StyledChevron = styled(ChevronDown, {
+  transition: "transform 300ms cubic-bezier(0.87, 0, 0.13, 1)",
```
Comment (wp-aberg) [inline ui/accordion/src/Accordion.tsx:124]: This easing gets repeated a bunch and the timing we have a token for. Could be re-written as 
```javascript
const easeInOutExpo = `cubic-bezier(0.87, 0, 0.13, 1)`;
...
transition: `transform ${theme.transitions.normal} ${easeInOutExpo}`
...
animation: `${slideDown} ${theme.transitions.normal} ${easeInOutExpo}`,
```

File: build.washingtonpost.com/docs/components/accordion.mdx (hunk 2 above)
Comment (atomictangerine) [reply build.washingtonpost.com/docs/components/accordion.mdx]: made it a bit smaller and also added a margin to the top of the two examples

File: ui/accordion/src/Accordion.tsx (hunk 3 above)
Comment (atomictangerine) [reply ui/accordion/src/Accordion.tsx:126]: good catch! thanks!

**Review by artmsilva — APPROVED** [review]

File: ui/accordion/src/Accordion.tsx [hunk 4]
```diff
@@ -0,0 +1,209 @@
+import * as React from "react";
+import { ChevronDown } from "@washingtonpost/wpds-assets";
+import { theme, styled, keyframes } from "@washingtonpost/wpds-theme";
+import { Icon } from "@washingtonpost/wpds-icon";
+import * as AccordionPrimitive from "@radix-ui/react-accordion";
+import type * as WPDS from "@washingtonpost/wpds-theme";
+import {
```
Comment (wp-aberg) [inline ui/accordion/src/Accordion.tsx:7]: We import these only to export them at the end. When the styled versions components get created the props come along with them without having to bring these along as well.

All that to say I think this is just extra complexity that we're not really getting anything from.

**Review by wp-aberg — APPROVED** [review]
Overall looks good to go, the import and export of Radix types is a little noisy but not a blocker.

**Pull request merged**